BitmovinYospacePlayerExample/BitmovinYospacePlayerExample/ContentView.swift

```diff
@@ -0,0 +1,177 @@
+import BitmovinYospacePlayer
+import BitmovinPlayerCore
+import SwiftUI
+
+// You can find your player license key on the player license dashboard:
+// https://bitmovin.com/dashboard/player/licenses
+private let playerLicenseKey = "<PLAYER_LICENSE_KEY>"
+
+private struct Stream: Hashable {
+    var title: String
+    var contentUrl: String
+    var fairplayLicenseUrl: String?
+    var fairplayCertUrl: String?
+    var drmHeader: String?
+    var yospaceSourceConfig: YospaceSourceConfig?
+    
+    static func == (lhs: Stream, rhs: Stream) -> Bool {
+        lhs.title == rhs.title && lhs.contentUrl == rhs.contentUrl
+    }
+    
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(title)
+        hasher.combine(contentUrl)
+    }
+}
```

Comment:
Why do you need Hashable?

```suggestion
private struct Stream {
    var title: String
    var contentUrl: String
    var fairplayLicenseUrl: String?
    var fairplayCertUrl: String?
    var drmHeader: String?
    var yospaceSourceConfig: YospaceSourceConfig?
}
```

Reply:
I needed to due to how I was handling stream selection. Looks like it's not needed anymore though 🙂